src/x/dice/dice.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
```

Comment:
I coulda sworn we already had a dice package that we use in a few different places. Is this just coming from m3x?

Reply:
This is from internal statsdex; I think there's a sampler package we have already, but it performs slightly differently (i.e. `NewSampler(0.1).Sample()` returns true IFF it's been called a multiple of 10 times)